client-gateway/bot/passenger.py

```
@@ -0,0 +1,212 @@
+
+
+def register_handlers(bot, user_orders, user_roles, buttons, keyboards, helpers):
+    
+    BTN_PASSENGER = buttons['BTN_PASSENGER']
+    BTN_ORDER_TAXI = buttons['BTN_ORDER_TAXI']
+    BTN_RATES = buttons['BTN_RATES']
+    BTN_SKIP = buttons['BTN_SKIP']
+    
+    passenger_menu = keyboards['passenger_menu']
+    skip_menu = keyboards['skip_menu']
+    role_selection_menu = keyboards['role_selection_menu']
```

Ruff (0.14.10)

Local variable `role_selection_menu` is assigned to but never used

Remove assignment to unused variable `role_selection_menu`

(F841)